apps/src/blockly/addons/contextMenu.js

```diff
@@ -249,6 +248,38 @@ const registerHighContrastTheme = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(highContrastThemeOption);
 };
 
+/**
+ * Change workspace theme to CdoAccessibleTheme
+ */
+const registerAccessibleTheme = function() {
+  const accessibleThemeOption = {
+    displayText: function(scope) {
+      return (
+        (isCurrentTheme(Themes.ACCESSIBLE, scope.workspace)
+          ? '✓ '
+          : `${msg.enable()} `) + msg.blocklyAccessibleTheme()
+      );
+    },
+    preconditionFn: function(scope) {
+      if (isMusicLabTheme(scope.workspace)) {
+        return MenuOptionStates.HIDDEN;
+      } else if (isCurrentTheme(Themes.ACCESSIBLE, scope.workspace)) {
+        return MenuOptionStates.DISABLED;
+      } else {
+        return MenuOptionStates.ENABLED;
+      }
+    },
+    callback: function(scope) {
+      localStorage.setItem(BLOCKLY_THEME, Themes.ACCESSIBLE);
+      scope.workspace.setTheme(Blockly.themes[Themes.ACCESSIBLE]);
+    },
+    scopeType: GoogleBlockly.ContextMenuRegistry.ScopeType.WORKSPACE,
+    id: 'accessibleTheme',
+    weight: 15
```

Comment:
Just curious - what is this `weight` value used for?

Reply:
`weight` is number that determines the sort order of the option. Options with higher weights appear later in the context menu. https://developers.google.com/blockly/guides/configure/web/context-menus

Reply:
Thanks!